README.md

```diff
@@ -466,5 +466,5 @@ the full [playlist] is on youtube
 1. [pip freezing (==) isn't safe (intermediate)](https://youtu.be/oGpyupM52IQ?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
 1. [python regex dynamic replacements (intermediate)](https://youtu.be/o_PD7AAbs4U?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
 1. [how I made my import sorter 7x faster (intermediate)](https://youtu.be/Q3menOSxRC4?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
-1. [what release was this fixed? (workflow) (intermediate) anthony explaine #458](https://youtu.be/7f3UdjuQYtw?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
+1. [what release was this fixed? (workflow) (intermediate)](https://youtu.be/7f3UdjuQYtw?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
```

Comment:
This seemed like a buggy title generation behavior, so I fixed it -- idk if you want it like this or not.